src/Sylius/Bundle/ResourceBundle/Controller/RequestConfiguration.php

```diff
@@ -257,7 +257,8 @@ public function getLimit()
         $limit = null;
 
         if ($this->isLimited()) {
-            $limit = (int) $this->parameters->get('limit', 10);
+            $limit= $this->parameters->get('limit', 10);
```

Comment:
```php
if (!$this->isLimited()) {
  return null;
}

$limit = $this->request->get('limit') ?: $this->parameters->get('limit');

return (int) $limit ?: 10;
```

WDYT? I'm not sure about the ternary operators path (is it more readable for someone else than me?) - I don't like using the second default parameter too many times. Although the early return path is surely a lot cleaner.